digester.go

```diff
@@ -338,6 +338,8 @@ func (d *sqlDigester) isLit(t token) (beLit bool) {
 		beLit = true
 	} else if t.lit == "*" {
 		beLit = true
+	} else if tok == null || (tok == identifier && strings.ToLower(t.lit) == "null") {
```

Comment:
``` `null` ``` should be normalized as ``` `null` ```, not `null`.

What it looks like in MySQL:
```sql
mysql> create table `null`(id int);
Query OK, 0 rows affected (0.04 sec)

mysql> select digest_text from performance_schema.events_statements_summary_by_digest where query_sample_text like 'create table `null`%';
+--------------------------------------+
| digest_text                          |
+--------------------------------------+
| CREATE TABLE `null` ( `id` INTEGER ) |
+--------------------------------------+
1 row in set (0.00 sec)
```

Reply:
Thank you for your comment. I agree with you. This would be fixed once #1151 is merged. So, it would be better to merge the present PR after #1151.